apps/wso2-digital-wallet/src/pages/Home/Home.css

```
@@ -7,270 +7,422 @@
+.recent-activity-topic {
+  font-size: 14px;
+  font-weight: 600;
+  color: #1C1917;
+  letter-spacing: -0.3px;
+  margin-bottom: 1px;
+}
+
+.recent-activity-address {
+  font-size: 11px;
+  color: #9CA3AF;
+  font-family: 'JetBrains Mono', 'Monaco', 'Menlo', 'Ubuntu Mono', monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)